py/picca/tests/delta_extraction/speed_tests.py

```diff
@@ -0,0 +1,155 @@
+"""This module contains speed tests that will be removed as deprecated
+functions are erased"""
+from time import time
+import timeit
+
```

Comment:
I guess this file will only stay inside the repo for this PR and be removed afterwards together with the slow options for find_bins/rebin

Reply:
I was planning on removing it before merging, yes. Just added in case you wanted to do some tests